modules/pico_ipv6.c

```diff
@@ -1883,6 +1894,11 @@ int pico_ipv6_link_del(struct pico_device *dev, struct pico_ip6 address)
     struct pico_ipv6_link test = {
         0
     }, *found = NULL;
+#ifdef PICO_SUPPORT_MCAST
+    struct pico_ip6 all_hosts = {{ 0xff, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01 }};
```

Comment:
Such a costant should really be somewhere else. maybe in one of the header files?